src/mobile/FooterInfo/FooterInfo.scss

```diff
@@ -0,0 +1,19 @@
+@import "../../scss/variables.scss";
+
+.footerInfo {
+  display: flex;
+  justify-content: space-between;
+
+  .copyright {
+    color: $primaryColorMain;
+    font-size: 21px;
+    line-height: 100%; /* 21px */
```

Comment:
They are inheritable css properties. You could move them (from here and `.mail` class) to `.footerInfo`.